statsmodels/regression/linear_model.py

```diff
@@ -2101,7 +2105,9 @@ def get_robustcov_results(self, cov_type='HC1', use_t=None, **kwds):
               `groups` : indicator for groups
               `time` : index of time periods
             - `maxlag` integer (required) : number of lags to use
-            - `kernel` string (optional) : kernel, default is Bartlett
+            - `kernel` callable or str (optional) : kernel
+                  currently available kernels are ['bartlett', 'uniform'],
+                  default is Bartlett
```

Comment:
Sure would be nice if this docstring were shared between here and the other location.